packages/client/src/checkout/types/postDraftOrders.types.ts

```diff
@@ -0,0 +1,15 @@
+import type { Config } from '../../types/index.js';
+import type { DraftOrder } from './draftOrder.types.js';
+
+export type PostDraftOrdersData = {
+  orderId: string;
+  customerId?: string;
```

Comment:
```suggestion
  customerId?: number;
```

Reply:
customerId is a string. You can see here: https://farfetch.atlassian.net/wiki/spaces/bw/pages/13525024822/spec54-checkout-draft-orders-managament-endpoints